### PR TITLE
Feature/add backbutton to end session in top left corner

### PR DESCRIPTION
**Pull request by lukas-menzel**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Functionality test in Google Chrome

**Test Configuration**:
* Go version:
* Kafka version:
* Kubernetes version:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: frontend/src/components/Result.js [hunk 1]
```diff
@@ -59,6 +60,11 @@ export class Result extends React.Component {
 
     return (
       <div className="body">
+        <div className="container">
+          <div className="column">
+            <BackButton />
```
Comment (reichhartd) [inline frontend/src/components/Result.js]:
```jsx
<div className="container column">
  <BackButton />
</div>
```

File: frontend/src/components/BackButton.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+function BackButton() {
+  return (
+    <div className="BackButton">
+      <a href="/">
+        <img className="BackButtonImg" src={backicon} width="24px" />
+      </a>
+    </div>
+  );
+}
```
Comment (reichhartd) [inline frontend/src/components/BackButton.js]: Less boilerplate code
```jsx
const BackButton = () => (
  <div className="BackButton">
    <a href="/">
      <img className="BackButtonImg" src={backicon} width="24px" />
    </a>
  </div>
);
```

File: frontend/src/components/Result.js [hunk 3]
```diff
@@ -111,13 +117,13 @@ export class Result extends React.Component {
         </div>
 
         <div className="container-plattforms">
-          <div classNme="column-one-sixed"></div>
+          <div className="column-one-sixed"></div>
```
Comment (reichhartd) [inline frontend/src/components/Result.js]:
```suggestion
          <div className="column-one-sixed" />
```
In general, I would advice you, to install [Prettier](https://prettier.io/), that does such things automatically and ensures a uniform codestyle.
In combination with [Pretty Quick](https://github.com/azz/pretty-quick) it runs as a pre-commit hook over the changed code, so you don't have to do it manually each time.

File: frontend/src/components/Result.js [hunk 4]
```diff
@@ -111,13 +117,13 @@ export class Result extends React.Component {
...
           <div className="column-four-sixed-posts">
             {posts.map(post => (
               <IGPost key={post.shortcode} post={post} />
             ))}
           </div>
-          <div classNme="column-one-sixed"></div>
+          <div className="column-one-sixed"></div>
```
Comment (reichhartd) [inline frontend/src/components/Result.js]:
```suggestion
          <div className="column-one-sixed" />
```

File: frontend/src/index.css [hunk 5]
```diff
@@ -155,6 +161,16 @@ button {
...
+.BackButton {
+  width: 40px;
+  height: 40px;
+  border: solid #000000 2px;
+}
+.BackButtonImg {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
```
Comment (reichhartd) [inline frontend/src/index.css:173]: Considering using [CSS Modules](https://github.com/css-modules/css-modules). When the project scales and you have a global CSS file defined, weird issues can happen, if you have defined the same class by mistake twice.

**Review by reichhartd — COMMENTED** [review]
No big issues, only a few code smells.

File: frontend/src/components/Result.js (hunk 1 above)
Comment (lukas-menzel) [reply frontend/src/components/Result.js]: changed it.

File: frontend/src/components/BackButton.js (hunk 2 above)
Comment (lukas-menzel) [reply frontend/src/components/BackButton.js]: changed it.

File: frontend/src/components/Result.js (hunk 3 above)
Comment (lukas-menzel) [reply frontend/src/components/Result.js]: I installed it already, but didn't activated it for the project. And I am also using Pretty Quick now. Thanks for the advise!

File: frontend/src/components/Result.js (hunk 4 above)
Comment (lukas-menzel) [reply frontend/src/components/Result.js]: changed it in all affected classes

File: frontend/src/index.css (hunk 5 above)
Comment (lukas-menzel) [reply frontend/src/index.css:173]: I will have a look into it and maybe implement it later on.

**Review by reichhartd — APPROVED** [review]

**Pull request merged**